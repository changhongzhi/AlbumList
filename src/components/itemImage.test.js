import React from 'react';
import renderer from 'react-test-renderer';
import ItemImage from './itemImage';

test('Show url img when show image', () => {
  const component = renderer.create(
    <ItemImage url="image url" showImage={true}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Show thumburl img when not show image', () => {
  const component = renderer.create(
    <ItemImage thumbnailUrl="thumbnail image url" showImage={false}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

