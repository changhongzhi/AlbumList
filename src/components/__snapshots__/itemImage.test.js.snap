exports[`test Show thumburl img when not show image 1`] = `
<img
  alt={undefined}
  src="thumbnail image url" />
`;

exports[`test Show url img when show image 1`] = `
<img
  alt={undefined}
  src="image url" />
`;
